html-block-and-inline/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h1>Blue Americana Snuggie&reg; Blanket</h1>
+      <h2>By SnuggieStore</h2>
+    </header>
+    <main>
+      <p>Price: <s>$19.99</s> $14.99</p>
+      <p>
+        Part of our new Americana collection, this <em>beautiful</em> blue is
+        already a customer favorite!
+      </p>
+      <p>Enjoy complete <strong>Freedom of Movement!</strong></p>
+      <p>
+        Does not slip and slide like a regular blanket and keeps you warm from
+        head to toe.
+      </p>
+    </main>
```

Comment:
Nicely done with implementing the `main` element!

Reply:
thanks :3